python/GangaDirac/Dirac.ini

```diff
@@ -1,6 +1,6 @@
 [Configuration]
 
-RUNTIME_PATH = GangaDirac
+RUNTIME_PATH = GangaGaudi:GangaDirac
```

Comment:
Just out of curiosity what part is pulling this in?

Reply:
This bit I'm not sure I'm going to include in the final PR yet, hence the "Note this is not quite ready to merge yet" however it is true that GangaDirac has a dependence on GangaGaudi which seems like it's only to do with simple functions. This leads me to think ultimately we should sever this link. If I run vanilla in a new cloud instance I see:

```python
Ganga In [1]: Job(backend=Dirac()).submit()
INFO     submitting job 0
INFO     job 0 status changed to "submitting"
INFO     Preparing Executable application.
INFO     Created shared directory: conf-2e87e008-3786-4c8e-a788-00a64ca3fc23
ERROR    ConfigError: attempt to create a configuration section [defaults_GaudiXMLSummary] after bootstrap 
ERROR    ConfigError: attempt to create a configuration section [defaults_GaudiXMLSummary] after bootstrap  ... reverting job 0 to the new status
INFO     job 0 status changed to "new"
ERROR    JobError: Error: ConfigError: attempt to create a configuration section [defaults_GaudiXMLSummary] after bootstrap 
```

So I must need to explicitly include the GangaGaudi plugin to get these config sections created at bootstrap time. However in testing im not sure that this does solve the problem so hence I don't think I'll be keeping this particular commit

Reply:
indeed the LHCb.ini includes:
```ini
[Configuration]
RUNTIME_PATH=GangaDirac:GangaGaudi:GangaLHCb
```

Reply:
whats concerning me is I'm sure I got DIRAC jobs running without this when setting up the docker images so maybe it's my cloud node.

Reply:
I didn't need this in the end.